Bottom: 1d6df7f1309de0f73e2d7a3b806b63bf7755ea15
Top:    61030beced3fc965ffdc83cf46540155409c937f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-23 02:56:09 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
